assets/src/edit-story/app/media/utils/getThumbnailUrl.js

```diff
@@ -23,17 +23,25 @@
  * @return {string} Source URL of the smallest available size image.
  */
 function getThumbnailUrl(minWidth, resource) {
-  if (resource.sizes) {
-    // The thumbnail is a square cropped image, we don't want that.
-    const sizesWithoutThumbnail = {
-      ...resource.sizes,
-    };
-    delete sizesWithoutThumbnail.thumbnail;
-    const isLandscape = (obj) => obj.width / obj.height > 1;
+  const getOrientation = (obj) => {
+    if (obj.width / obj.height > 1) {
+      return Orientation.LANDSCAPE;
+    } else if (obj.width / obj.height < 1) {
+      return Orientation.PORTRAIT;
+    }
+    return Orientation.SQUARE;
+  };
 
-    const smallestValidImage = Object.values(sizesWithoutThumbnail)
+  const Orientation = {
+    PORTRAIT: 'portrait',
+    LANDSCAPE: 'landscape',
+    SQUARE: 'square',
```

Comment:
Was it not enough for 'square' to evaluate as 'PORTRAIT' for both 's' and 'resource' in the filter?

Reply:
That might return a square thumbnail for a portrait image, which would stretch it in the opposite direction.